### DISCUSSION TITLE
Help with NuGet

### DISCUSSION THREAD

**Discussion by SuperWasTaken** [CosmosOS/Cosmos#2791] [Q&A] [answered]
Hello I am having some problems with NuGet I want to install a thrid party module and I'm getting this error when I am trying to install it anyone know what's happening?

![image](https://github.com/CosmosOS/Cosmos/assets/145633746/c6de257a-c5dc-4131-88f2-4767ce5a6ae6)

**Comment by ascpixi** [comment, marked as answer]
It looks like you're using the user-kit, while Cosmos.Coroutines was compiled for the latest dev-kit. Try installing the dev-kit and upgrading your project to use it instead, or, if you still want to use the user-kit, you can always choose to [compile the project yourself.](https://github.com/ascpixi/cosmos-coroutines)

**Reply by SuperWasTaken** [reply]
How do I compile the library myself and import it.

**Reply by ascpixi** [reply]
It should be straight-forward; clone the repo, and then either include the project in your solution, and reference it from your C# project, or build it and include the assembly (DLL) as a reference instead. You might have to adjust the `.csproj` file of the project to use the specific version of the toolkit you have.

**Reply by SuperWasTaken** [reply]
Thanks I will look into that later

**Reply by 9xbt** [reply]
It would be easier to upgrade to the devkit though

**Reply by SuperWasTaken** [reply]
Can the Dev Kit be used with VS 2022 because If not I'm sadly stuck on the user kit unless if I want to restart my entire project

**Reply by ascpixi** [reply]
The API changes from the user-kit to the dev-kit aren't too drastic. You should be able to easily change your project to use it instead (user kits are merely snapshots of the main branch, which are deemed as "release-ready")
